### ISSUE TITLE
How to retrieve hiddendata

### ISSUE THREAD

**Issue by dhalul** [wumke/react-native-local-notifications#22]
I see in the readme we can provide data to createNotification/updateNotification parameter.
How to get that hiddendata from rn component?

Thanks

**Comment by wumke** [comment]
I use this package in combination with the react-native-push-notification package, their 'onNotification(notification)' function triggers also when the app is opened from a notification. I check there if notification.userInteraction is true, then get data for direct navigation to the specific page and content by getting it from the notification this way: 
```
            if(platform === 'and'){
                hiddendata = notification.hiddendata;
            }
            else {
                hiddendata = notification.data.hiddendata
            }
```
As you can see it is just an additional field in the data (which you must get directly for android and via .data for ios).
Some day, when I find the time, I want to integrate this directly in this library... for now I suggest you use the push-notification package for this purpose.

**Comment by dhalul** [comment]
So what you mean is i still using this package and then i install the react-native-push-notification package, and i just use onNotification function in PushNotification.configure to receive the data ?

**Comment by wumke** [comment]
Yes, I think that should work, maybe you need to provide some other fake data in the configure object. Feel free to try and post your findings here.
